dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/DataPointBaseInformation.kt

```diff
@@ -7,13 +7,11 @@ import org.dataland.datalandbackend.model.enums.data.QualityOptions
  * --- API model ---
  * Fields of a generic data point and its source
  */
-data class DataPoint<T>(
-    val value: T? = null,
-
+open class DataPointBaseInformation(
     @field:JsonProperty(required = true)
```

Comment:
I think you don't need the json annotation here.
It is sufficient if the data class of the DataPoint-model which extends this base class has the correct annotation at the required fields (e,g, see "SmeProduct.kt" and "ProductBase.kt" as an example)

Reply:
Trying it out

Reply:
Removed

Reply:
👍